SPECS/dracut/dracut.spec

```diff
@@ -135,6 +132,7 @@ install -m 0644 dracut.conf.d/fips.conf.example %{buildroot}%{_sysconfdir}/dracu
 > %{buildroot}%{_sysconfdir}/system-fips
 
 install -m 0644 %{SOURCE3} %{buildroot}%{_sysconfdir}/dracut.conf.d/50-megaraid.conf
+install -m 0644 %{SOURCE6} %{buildroot}%{_sysconfdir}/dracut.conf.d/00-defaults.conf
```

Comment:
Just curious, is 00-defaults.conf the right importance/ordering value to use here?

Reply:
No idea. Considering that these are default options for the image that can be overridden by the user, `00` made sense to me.